assessments/guide/security-reviewer.md

```diff
@@ -43,4 +43,35 @@ expected to conduct an in-depth review described in the [Security Assessment
 Guide](./). Security Reviewers should seek the approval of the other
 participating Security Reviewers and at least 1 co-chair before merging.
 
+## Conflict of interest
 
+There is a possibility of conflict of interest that can arise between a security reviewer and
+the project being reviewed due to the closely-knit nature of the community. Having clear
+guidelines for conflict of interest situations are important to prevent:
+
+- Individuals from intentionally or unintentionally promote their own company's project
+- SIG-Security chairs and assessment leads could intentionally or unintentionally limit the participation of an individual unfairly by asserting conflict of interest
+- Security reviews being stalled while groups discuss whether this person or that person should be allowed to participate
```

Comment:
Just a nit
Instead of *discuss whether this person or that person* how about "groups belabor on who"